content/en/docs/advanced-tasks/communicating-operator-conditions-to-olm.md

```diff
@@ -0,0 +1,82 @@
+# Communicating Operator Conditions to OLM
+
+As part of its role in managing the lifecycle of an operator, the Operator-Lifecycle-Manager (OLM) infers the state of an operator from the state of Kubernetes resources that define the operator. While this approach provides some level of assurance that an operator is in a given state, there are many instances where an operator may wish to communicate information to OLM that could not be inferred otherwise. This information can then be used by OLM to better manage the lifecycle of the operator.
+
+## OperatorConditions
+
+OLM introduced a new [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) called the [OperatorCondition](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_operatorconditions.yaml) allowing operators to communicate conditions to OLM. There are a set of "OLM Supported Conditions" which influence OLM's management of the operator when present in the OperatorCondition's [Status.Conditions](https://github.com/operator-framework/api/blob/b55a341f6560db4adec39d69aab1ff3092ea202a/pkg/operators/v1/operatorcondition_types.go#L22) array.
```

Comment:
We should probably include the CRD [here](https://github.com/operator-framework/olm-docs/tree/master/content/en/docs/Concepts/crds) instead of pointing to the codebase.

Reply:
nit: There are a set of "OLM Supported Conditions": did you mean for the OLM Supported Conditions to be in bold?

Reply:
>nit: There are a set of "OLM Supported Conditions": did you mean for the OLM Supported Conditions to be in bold?

No, I'm using it as a term to refer to a subset of conditions

Reply:
>We should probably include the CRD here instead of pointing to the codebase.

Done.